docs/en/03_How_tos/external_http_requests_with_proxy.md

```diff
@@ -3,26 +3,35 @@ summary: How to access external resources via the CWP proxy.
 
 # External HTTP requests with proxy
 
-All CWP environments are positioned behind a proxy. All requests out to external services must go through this proxy.
+All requests to resources outside of your stack must use a proxy. CWP environments have no direct HTTP connectivity out to the internet nor to other CWP stacks.
```

Comment:
@mateusz Could we say ...nor to other CWP stacks *internally without using the proxy*? 
It would clarify that you can connect to the stacks if you use the proxy.

Reply:
The sentence didn't flow well, but have reworded to:

> All requests to resources outside of your stack must use a proxy. CWP environments have no direct, unproxied HTTP connectivity out to the internet nor to other CWP stacks internally.